python/datasources/covid_tracking_project_metadata.py

```diff
@@ -10,3 +11,48 @@ def get_id():
     @staticmethod
     def get_table_name():
         return 'covid_tracking_project_metadata'
+
+    def upload_to_gcs(self, _, **attrs):
+        raise NotImplementedError(
+            'upload_to_gcs should not be called for CtpMetadata')
+
+    def write_to_bq(self, dataset, gcs_bucket, **attrs):
+        gcs_file = self.get_attr(attrs, 'filename')
+
+        # Download the raw data
+        df = gcs_to_bq_util.load_csv_as_dataframe(gcs_bucket, gcs_file)
+        self.clean_frame_column_names(df)
+
+        # Standardize the data
+        keep_cols = [
+            'state_postal_abbreviation', 'api_death',
+            'defines_other_death', 'race_ethnicity_separately_death',
+            'race_ethnicity_combined_death', 'race_mutually_exclusive_death',
+            'combined_category_other_than_api_death', 'race_death',
+            'ethnicity_death', 'api_cases', 'defines_other_cases',
+            'race_ethnicity_separately_cases', 'race_ethnicity_combined_cases',
+            'race_mutually_exclusive_cases', 'combined_category_other_than_api_cases',
+            'race_cases', 'ethnicity_cases']
+        df = df[keep_cols]
+        df = df.melt(id_vars=['state_postal_abbreviation'])
+        df[['col_name', 'variable_type']] = df.variable.str.rsplit(
+            '_', 1, expand=True)
+        df.drop('variable', axis=1, inplace=True)
+        df = df.pivot(
+            index=['state_postal_abbreviation', 'variable_type'],
+            columns='col_name', values='value').reset_index()
+        df.rename_axis(None, inplace=True)
+        df.rename(columns=self._metadata_columns_map(), inplace=True)
+
+        # Write to BQ
+        gcs_to_bq_util.append_dataframe_to_bq(df, dataset, self.get_table_name())
+
+    @staticmethod
+    def _metadata_columns_map():
+        """Returns a dict for renaming raw column names."""
+        return {
+            'api': 'reports_api',
+            'combined_category_other_than_api': 'reports_ind',
+            'ethnicity': 'reports_ethnicity',
+            'race': 'reports_race'
```

Comment:
Not sure what the ideal place to document these columns is, but doesn't hurt to briefly summarize them in the code

Reply:
Detailed definitions for these names are in the Structured Metadata for Covid Tracking Project doc. Ideally, that doc could get turned into a wiki and we could link to it from here.